src/nav-bar/nav-links.md

```diff
@@ -0,0 +1,50 @@
+---
+template: nav-links
+nav_items:
+  - label: Earn
+    isExternal: false
+    subitems:
+      - label: How to Earn
+        url: /earn
+      - label: Staker
+        url: earn/staker
+      - label: Liquidity Provider
+        url: earn/liquidity-provider
+      - label: BTC/tBTC User
+        url: /earn/btc
+      - label: Token Holder
+        url: /earn/token-holder
+  - label: Governance
+    url: /governance
+  - label: News
+    subitems:
+      - label: Press
+        url: /press
+      - label: Blog
+        url: https://blog.threshold.network/
+        isExternal: true
+  - label: About
+    subitems:
+      - label: Contributors
+        url: /contributors
+      - label: FAQ
+        url: /faq
+      - label: Audits
+        url: /audits
+social_links:
+  - label: Twitter
+    url: https://twitter.com/TheTNetwork
+    icon:
+      image: /images/twitter.svg
+      alt: Thershold Twitter
+  - label: Discord
+    url: https://discord.gg/VafbhzJ3vW
+    icon:
+      image: /images/discord.svg
```

Comment:
Looks like we'll need to use only local icons and images with the CMS? Did you just copy this SVG from react-icons?

```
import { FaTwitter, IoLogoDiscord, IoLogoGithub } from "react-icons/all"
```

Reply:
I exported icons from the Figma.